src/web/app/pages-instructor/instructor-course-edit-page/instructor-course-edit-page.component.ts

```diff
@@ -907,17 +907,16 @@ export class InstructorCourseEditPageComponent implements OnInit {
     const paramsMap: { [key: string]: string } = {
       courseid: this.courseToEdit.id,
       instructorid: this.instructor.googleId,
-      instructoremail: instructorToDelete.email,
-      instructorisdisplayed: instructorToDelete.isDisplayedToStudents.toString(),
     };
 
-    this.httpRequestService.delete('/instructors/course/details/deleteInstructor', paramsMap)
-        .subscribe((resp: MessageOutput) => {
+    this.httpRequestService.delete('/instructor', paramsMap)
+        .subscribe(() => {
           if (instructorToDelete.googleId === this.instructor.googleId) {
-            this.navigationService.navigateWithSuccessMessage(this.router, '/web/instructor/courses', resp.message);
+            this.navigationService.
```

Comment:
Please break the line at the right place.
```suggestion
            this.navigationService.navigateWithSuccessMessage(
                    this.router, '/web/instructor/courses', 'Instructor is successfully deleted.')
```